test/apiv2/12-imagesMore.at

```diff
@@ -86,6 +86,33 @@ t POST "libpod/images/scp/$IMAGE?destination=QA::" 500 \
 t DELETE libpod/images/$IMAGE 200 \
   .ExitCode=0
 
+# list images through the libpod endpoint should not return
+# IsManifestList (bool), Arch (string), and Os (string)
+podman pull -q $IMAGE
+t GET libpod/images/json 200  \ .[0].IsManifestList=false \
+                                .[0].Arch=amd64 \
+                                .[0].Os=linux
+podman rmi $IMAGE
+
+# if an image is a manifest list, it should not have
+# anything for arch or os
+podman manifest create foobar
+t GET libpod/images/json 200  \ .[0].IsManifestList=true \
+                                .[0].Arch=null \
+                                .[0].Os=null
+podman manifest rm foobar
+
+# if a manifest list and an image are returned with libpod images
+# endpoint, then one should be a manifest with IsManifest only; and
+# the other image should have IsManifestList, Arch, and Os.
+podman manifest create foobar
+podman pull -q $IMAGE
+podman manifest add --arch amd64 foobar $IMAGE
+t GET libpod/images/json 200  \ .[0].IsManifestList=true
```

Comment:
Nuke the midline backslashes please. Backslash means "to be continued on next line", and is only meaningful at EOL